src/autolaser.py

```diff
@@ -112,13 +112,21 @@ def draw_outline(max_contour, frame):
     ((x, y), radius) = cv2.minEnclosingCircle(max_contour)
 
     center = find_center_point(max_contour)
-
-    # Find the distance between the balloon and the camera
-    distance = distanceFinder(focalLength, KNOWN_RADIUS, radius)
     
+     # Find the distance between the balloon and the camera
+    distance = distanceFinder(focalLength, KNOWN_RADIUS, radius)
+
+    ft_distance = round(int(distance) / 12, 1)
+
     if radius > 10:
         cv2.circle(frame, center, 5, (0, 0, 255), -1)
         cv2.drawContours(frame, [max_contour], 0, (0, 255, 0), 3)
+        distance_text = "Distance (FT): " + str(ft_distance)
+    else:
+        distance_text = "Distance (FT): N/A"
+
```

Comment:
made it so we always have the display there rather than popping in and out which could be sort of jarring

Reply:
Looks good